utility_code/export_loot_table_items.py

```diff
@@ -181,8 +193,23 @@ def get_refined_creative_tab_flags(item_tag):
             if DEBUG and DEBUG_ITEM_CONTAINS in item_name:
                 print(f'Finally, {item_name} has been declared {out_dict["release_status"]}')
 
+# Filter the actual json output map that gets written based on the release status filter specified on the command line
+json_out_map = {}
+for item_type in out_map:
+    item_type_out = {}
+    out_types = out_map[item_type]
+    for item_name in out_types:
+        item_entry = out_types[item_name]
+        print(item_entry.get("release_status", ""))
```

Comment:
I'd be tempted to assign this to a variable instead of checking it more than once, but this works fine.

Reply:
Oh, I forgot to remove this print